ad9959/src/lib.rs

```diff
@@ -530,7 +530,7 @@ impl ProfileSerializer {
     pub fn new(mode: Mode) -> Self {
         Self {
             mode: mode as _,
-            data: [0; 32],
+            data: [0; 64],
```

Comment:
No. Why?

That's more than the qspi fifo.

Instead just recreate the builder and wait/delay for flush.

Reply:
Ok.

Should I keep track of the DDS parameters (or read back), then only update when necessary?

Reply:
AFAICT frequency/phase/amplitude all update without transients. Writing them always should be fine. But may well be worth checking on hardware.
NB: Changing the max buffer length would also require changes to the IO_UPDATE timing and would break assumptions about the max duration of such an update elsewhere.

Reply:
> Instead just recreate the builder and wait/delay for flush.
Done. The QSPI enters indirect write mode after `DdsOutput` initializes and starts streaming. So when writing to a full FIFO, the write operation will stall until there are enough space for the data.